content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss

```diff
@@ -98,6 +98,10 @@ $excerpt-line-height: 20;
     .excerpt {
       // show only 3 lines of copy
       @include limit-visibile-lines(3, $excerpt-line-height, $excerpt-font-size);
+
+      @at-root .ds-card-grid-divisible-by-3 & {
```

Comment:
may be add a line here what this does or what it compiles to

Reply:
`@at-root` is part of Sass, but I can document it if feels necessary :)